llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -2947,6 +2947,8 @@ bool AddressSanitizer::instrumentFunction(Function &F,
   if (F.getLinkage() == GlobalValue::AvailableExternallyLinkage) return false;
   if (!ClDebugFunc.empty() && ClDebugFunc == F.getName()) return false;
   if (F.getName().starts_with("__asan_")) return false;
+  if (F.isPresplitCoroutine())
+    return false;
```

Comment:
nit: for consistency with preceding lines:

```suggestion
  if (F.isPresplitCoroutine()) return false;
```

Reply:
They were on the same line, and I don't see a problem with that (see the first commit), but clang-format complained.